src/main/kotlin/no/nav/familie/ef/sak/api/dto/DokumentinfoDto.kt

```diff
@@ -0,0 +1,12 @@
+package no.nav.familie.ef.sak.api.dto
+
+import no.nav.familie.kontrakter.felles.journalpost.Journalposttype
+import java.time.LocalDate
+
+data class DokumentinfoDto(
```

Comment:
Jeg har akkurat gått gjennom hele sak og fjernet alle forkomster av linjer som slutter med (, for å prøve å ha samme stil på all kode. Ikke innfør nye er du snill.

Reply:
Må jo lage litt forvaltningsarbeid til senere 😂